app/src/main/java/com/example/riden/Adapters/RideAdapter.java

```diff
@@ -18,11 +23,15 @@
 import com.example.riden.models.User;
 import com.parse.ParseFile;
 
+import java.sql.Array;
+import java.util.ArrayList;
+import java.util.Collection;
 import java.util.List;
 
-public class RideAdapter extends RecyclerView.Adapter<RideAdapter.ViewHolder> {
+public class RideAdapter extends RecyclerView.Adapter<RideAdapter.ViewHolder> implements Filterable {
     private Context context;
     private List<Ride> rides;
+    private List<Ride> allRides;
     private TextView tvDestination;
```

Comment:
These need to be moved to ViewHolder right, is that done in another diff ? Let's just add it to another iteration of this diff